phpunit.xml.dist

```diff
@@ -28,6 +28,7 @@
 				<directory suffix=".php">./language/</directory>
 				<directory suffix=".php">./migrations/</directory>
```

Comment:
Why don't process these?

Reply:
Because these files are outside the scope of what we are actually unit-testing (they still get used during tests, but PhpUnit won't load/scan them during its initialization). There's nothing to test directly in the `language` or `migration` folders, as they are just data arrays. And we can't test the `tests` - I mean the `tests` test themselves just by being run, lol.

The `vendor` folder has to be excluded too, otherwise it causes fatal errors by trying to redeclare the autoload class (it was already declared in phpBB's vendor dir). This error pops up during unit-testing with code coverage, but not testing w/o code coverage. But same thing applies here, we're not unit-testing anything in the vendor dir since that's all 3rd party software and someone else's job to test, lol.

Reply:
Well, What I've wondered about is just `migration` and `language` folders as they contain code at least to lint etc. :)
Okay.